addon/queue.js

```diff
@@ -148,6 +148,10 @@ export default Ember.Object.extend({
     let allFilesHaveSettled = files.every(function (file) {
       return ['uploaded', 'aborted'].indexOf(file.state) !== -1;
     });
+    
+    if (files.length <= 0) {
```

Comment:
Could we change this to `files.length === 0`?

I don't know how we could have an array with less than 0 items in it.